Makefile.am

```diff
@@ -508,10 +508,13 @@ cppcheck:
 
 dist-hook: $(top_distdir)/configure
 	@if grep 'pkg_m4_absent' '$(top_distdir)/configure'; then \
-	   echo 'configure is generated without pkg.m4. Please supply pkg.m4 and run ./autogen.sh to rebuild the configure script.'>&2; \
+	   echo 'ERROR: configure is generated without pkg.m4. Please supply pkg.m4 and run ./autogen.sh to rebuild the configure script.'>&2; \
 	   (exit 1); \
 	else :; \
 	fi
+	@if grep 'PACKAGE_VERSION.*-g' '$(top_distdir)/configure'; then \
+	  echo 'WARNING: You are building a dist from a git version. Better run make dist outside of a .git repo on a tagged release.'>&2; \
```

Comment:
I didn't get what the warning is for. What's wrong with doing `make dist` inside a git repo?
For what I've seen is that `make dist` shouldn't touch anything in the source directory.

Reply:
There's nothing wrong which but it's likely intent you want tagged releases for a dist tarball and not your random git commit.